.travis.yml

```diff
@@ -46,7 +47,7 @@ before_install:
   galaxy_kickstart`
 - docker ps
 install:
-- sudo su $GALAXY_TRAVIS_USER -c 'pip install --user bioblend[testing]'
+  - sudo su $GALAXY_TRAVIS_USER -c 'pip install --user https://github.com/mvdbeek/bioblend/archive/test_data.zip pytest'
```

Comment:
OK, I see. This `pip install --user bioblend[testing]` will cause the extra dependencies of bioblend to be installed (see https://github.com/galaxyproject/bioblend/blob/master/setup.py#L40). But this syntax doesn't work when you point pip to a url, hence the explicit installation of pytest.